R/plotting.R

```diff
@@ -510,72 +620,98 @@ plotMotifHeatmaps <- function(x,
 
 #' @title Plot Stability Paths
 #'
-#' @description Plot the stability paths of each variable (predictor), 
+#' @description Plot the stability paths of each variable (predictor),
 #'   showing the selection probability as a function of the regularization step.
 #'
-#' @param se the \code{SummarizedExperiment} object resulting from stability 
+#' @param se the \code{SummarizedExperiment} object resulting from stability
 #'   selection, by running \code{\link[monaLisa]{randLassoStabSel}}.
 #' @param selProbMin A numerical scalar in [0,1]. Predictors with a selection
 #'   probability greater than \code{selProbMin} are shown as colored lines. The
 #'   color is defined by the \code{col} argument.
-#' @param col color of the selected predictors.
-#' @param lwd line width (default = 1).
-#' @param lty line type (default = 1).
-#' @param ylim limits for y-axis (default = c(0,1.1)).
-#' @param ... additional parameters to pass on to \code{matplot}.
-#'
-#' @return \code{TRUE} (invisibly).
-#' 
-#' @examples 
+#' @param col vector of colors specifying what to color selected and
```

Comment:
This sentence is a bit hard to parse. Perhaps can also clarify that it should be a named vector with names 'sel' and 'notSel'

Reply:
good point! I was wondering if there was a more elegant way to do this as well.

Reply:
I think the other option might be to just have two arguments - e.g. `selColor` and `notSelColor` or something like that, and let the user specify these directly.

Reply:
yes that nicer!